src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -114,25 +115,27 @@ class SearchControls extends Component {
     return (
       <div className="search-controls-container clearfix">
         <form className="row" onSubmit={this.handleSubmit}>
-          <div className="columns medium-4">
-            <label
-              htmlFor="street-city-state-zip"
-              id="street-city-state-zip-label"
-            >
-              Enter Street, City, State or Zip
+          <div className="columns medium-3-5">
```

Comment:
I'm not a fan of hacking the Foundation grid. What was happening before that redefining the widths was necessary? If it's super important, I would rename these so it doesn't get mistaken for real Foundation grid classes.

Reply:
As I recall this had to do with the fact that "Community Care (Non-VA Health)" was long enough that it was reflowing the entire search bar to a second line. Our first approach had the Search button being shrunk, but to bring the button back up to a larger size this was done.

I will direct the naming choices question to @CLChilton as he helped with the CSS changes here.

Reply:
Hi, I named them like they were extensions or sub-classes of the original grid definitions. I made them to get a little more fine-grained width control of the containers. I don't know if they're super-important or not but we can rename them if needed.